src/cryptography/x509/extensions.py

```diff
@@ -1664,14 +1711,14 @@ class IssuingDistributionPoint(ExtensionType):
 
     def __init__(
         self,
-        full_name,
-        relative_name,
-        only_contains_user_certs,
-        only_contains_ca_certs,
-        only_some_reasons,
-        indirect_crl,
-        only_contains_attribute_certs,
-    ):
+        full_name: typing.Optional[typing.List[GeneralName]],
```

Comment:
Special note here: We never cast full_name to a list (as we do in many places). So you really should pass a list - at present. Should we do `list(full_name) and use `Iterable`  instead?

Reply:
We should cast and use iterable! If mypy is helping us see inconsistency let's stamp it out 😄